packages/opentelemetry/src/asyncContextStrategy.ts

```diff
@@ -102,9 +103,10 @@ export function setOpenTelemetryContextAsyncContextStrategy(): void {
     startSpanManual,
     startInactiveSpan,
     getActiveSpan,
+    suppressTracing,
     // The types here don't fully align, because our own `Span` type is narrower
     // than the OTEL one - but this is OK for here, as we now we'll only have OTEL spans passed around
     withActiveSpan: withActiveSpan as typeof defaultWithActiveSpan,
-    suppressTracing: suppressTracing,
+    getTraceData: getTraceData as typeof defaultGetTraceData,
```

Comment:
is this type cast needed here? It should not _generally_ be needed, only when the method accepts spans, I think?

Reply:
nevermind, I see it _does_ take a span xD